packages/firebase_admob/android/src/main/java/io/flutter/plugins/firebaseadmob/MobileAd.java

```diff
@@ -219,4 +233,47 @@ void show() {
 
     // It is not possible to hide/remove/destroy an AdMob interstitial Ad.
   }
+
+  static class Native extends MobileAd {
+    private UnifiedNativeAd nativeAd;
+
+    private Native(int id, Activity activity, MethodChannel channel) {
+      super(id, activity, channel);
+    }
+
+    @Override
+    void load(String adUnitId, Map<String, Object> targetingInfo) {
+      status = Status.LOADING;
+
+      final AdLoader adLoader = new AdLoader.Builder(activity, adUnitId)
+          .forUnifiedNativeAd(new UnifiedNativeAd.OnUnifiedNativeAdLoadedListener() {
+            @Override
+            public void onUnifiedNativeAdLoaded(UnifiedNativeAd unifiedNativeAd) {
```

Comment:
NativeAds don't call `onLoaded` like all other Ads. They call this method instead because it passes the user a `UnifiedNativeAd`. 

Since the information in `UnifiedNativeAd` is used to create a View is only useful on the Platform side, I have this Ad call the normal `onLoaded` method.